src/meetings/2018-03-13/index.md

```diff
@@ -1,44 +1,9 @@
 ---
-meeting:
-  date: 2018-03-13
-  time:
-    start: 6:00pm
-    end: 7:00pm
-host:
-  location: Gravitate, 317 6th Ave, Suite 10
-  city: Des Moines
-  state: IA
-  zip: 50309
-sponsor:
-  name: Robert Half
-  site: https://www.roberthalf.com/ia-des-moines
-talk: 
-  title: An Introduction to Transducers
-  tags: Functional Programming
-  speaker: Eric Ponto
-  twitter: ericponto
-  slides: http://slides.ericponto.com/dsmjs-intro-to-transducers/
+date: 2018-03-13
+time:
+  start: 6:00pm
+  end: 7:00pm
+host: Gravitate
```

Comment:
So this basically ties to `src/hosts/gravitate-downtown.md` and up to the key/value of `location: Gravitate`?

Reply:
yep, the actual mappings are defined here: https://github.com/dsmjs/site/blob/322a6c515c94bdce6b3933f26373972509a1bbee/gatsby-config.js#L6-L9